### PR TITLE
set paths for FP OS correctly

### PR DESCRIPTION
**Pull request by mad-de**

### CODE REVIEW

File: extract-files.sh [hunk 1]
```diff
@@ -15,9 +15,12 @@ mkdir -p $BASE/$DIR
 adb pull /system/$FILE $BASE/$FILE
 done
 
-# some extra stuff
+# Pull extra stuff from FP OS
 adb pull /system/lib/hw/audio_policy.default.so $BASE/lib/hw/audio_policy.mt6589.so
-adb pull /system/lib/libaudio.primary.default.so $BASE/lib/hw/audio.primary.mt6589.so
+adb pull /system/lib/hw/libaudio.primary.default.so $BASE/lib/hw/audio.primary.mt6589.so
```
Comment (z3ntu) [inline extract-files.sh]: in `/system/lib/hw` to only files are

> -rw-r--r-- root     root        13604 2008-08-01 14:00 libaudio.r_submix.default.so
> -rw-r--r-- root     root         9488 2008-08-01 14:00 libaudio.usb.default.so

, BUT `libaudio.primary.default.so` is in `/system/lib`

**Comment by mad-de** [comment]
You said earlier:

File: extract-files.sh (hunk 1 above)
Comment (mad-de) [inline extract-files.sh]: You said earlier:

shell@android:/system/lib/hw # ls -al
-rw-r--r-- root root 5348 2008-08-01 14:00 audio.primary.default.so
-rw-r--r-- root root 136952 2008-08-01 14:00 audio_policy.default.so
-rw-r--r-- root root 5292 2008-08-01 14:00 audio_policy.stub.so
and more files

so these files aren't in /system/lib/hw ?

Comment (z3ntu) [inline extract-files.sh]: only the `libaudio.primary.default.so` is ŃOT in the `hw` folder

Comment (mad-de) [inline extract-files.sh]: Sorry I didn't got that the first time. Can you confirm that its fixed and working with https://github.com/mad-de/android_device_fp_FP1/commit/71350406fb64d908c2f8ed1829da10cc025d16b6 ?

**Pull request merged**